draft-ietf-radext-radiusdtls-bis.md

```diff
@@ -749,6 +751,20 @@ The proxy can do RADIUS/UDP to some servers and RADIUS/(D)TLS to others.
 Delegation of responsibilities and separation of tasks are important security principles.
 By moving all RADIUS/(D)TLS knowledge to a (D)TLS-aware proxy, security analysis becomes simpler, and enforcement of correct security becomes easier.
 
+## Crypto-Agility
+
+Crypto-agility requirements were discussed extensively in {{?RFC6421}}. {{RFC6614, Appendix C}} described how those requirements were met for RADIUS/TLS and {{RFC7360, Section 10.1}} describes the same for RADIUS/DTLS. For clarity, the final outcome of the those discussions are repeated, using the text of {{RFC7360}} with some minor modifications to update references, without changing the content.
```

Comment:
```suggestion
Crypto-agility requirements for the RADIUS protocol were discussed extensively in {{?RFC6421}}. {{RFC6614, Appendix C}} described how those requirements were met for RADIUS/TLS and {{RFC7360, Section 10.1}} describes the same for RADIUS/DTLS. For clarity, the final outcome of the those discussions are repeated, using the text of {{RFC7360}} with some minor modifications to update references, without changing the content.
```

It's not "crypto-agility in general" but "crypto-agility for RADIUS"
We should make that clear.

Reply:
Perhaps add a ref to https://datatracker.ietf.org/doc/html/draft-ietf-radext-deprecating-radius-05, which discusses crypto agility